### ISSUE TITLE
[EN] [Mechanic] [Warrior] Victory Rush w/ Gurthalak

### ISSUE THREAD

**Issue by Heroclassx** [Atlantiss/BugTracker#10191]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: https://www.wowhead.com/spell=122979/victory-rush - Can only be used within 20 sec after you *kill an enemy that yields honor or experience. 

**Current behaviour**: When Gurthalak tentacles are killed the Warrior gets Victory rush.

**Expected behaviour**: Gurthalak tentacles neither reward honor nor exp => no victory rush should proc on kill. This is a big issue in PvP snice warrior heals more of tentacles then they deal damage. 

**Steps to reproduce the problem**:

1. Hit warrior till gurthalak procs
2. Warrior kills tentacles
3. Victory Rush procs
4. unequip gurthalak when fighting a warrior

**Revision**: Current server version (you can see current server revision by typing ".server info" command in the game chat)

**Comment by Piklo** [comment]
so hunters pet reward honor / exp?
https://github.com/Atlantiss/BugTracker/issues/10188

**Comment by TranceAccount** [comment]
Prooof

**Comment by Heroclassx** [comment]
Yeah I've seen that report, well idk, maybe blizzard bug? Tooltip says only targets which yield exp+honor nor is happening on tentacles & hunter pets

**Comment by Heroclassx** [comment]
Well tbh, since tooltip says so imo it should be other way around, there should be a proof that it is working AGAINST the tooltip :D But I will try to find a video or some post about it when I am home.

**Comment by Illusie** [comment]
I remember in several broken servers and lolten you could get VR from Hunter's pet and Snakes but that was like 4-5y ago : D

VR from tentacle is nojs

**Comment by keikox** [comment]
you should get victory rush proc from killing hunter's pet, that's how it was on retail and numerous other cata servers. however it shouldn't proc from killing tentacle or warlock/mage pet.

**Comment by MangeNerfedOwl** [comment]
Same problem with guardians.
